Files/powerGate/Modules/Communication.psm1

```diff
@@ -2,12 +2,31 @@
 Import-Module powerGate
 
 #TODO: configure the powerGate server url and port
-$powerGateServerName = $ENV:Computername
+$powerGateServerName = getRelatedPGServerName
 $powerGateServerPort = "8080"
+$vaultDefinitions = @{
+	"TEST" = "TestVault";
```

Comment:
Since this is a configuration mapping here. 
**ToDo**: Please add comments that not the scripter know what they need to configure here.